### ISSUE TITLE
Keyboard navigation doesn't work intuitively in 'about us timeline' slice

### ISSUE THREAD

**Issue by samsmith1983** [redbadger/website-honestly#321]
Repro:

- Go to /about-us on Chrome
- Tab through the page

When you reach the 'about us timeline' slice, you won't see any highlighting for a few tab presses.

**Comment by Roisi** [comment]
Slightly wider issue as the tabbability of the whole page isn't great.
